src/test/web/index.ts

```diff
@@ -54,6 +60,8 @@ export async function activate(context: IExtensionContext): Promise<IExtensionAp
             }
         });
     });
+
+    // Can't tell if on CI or not, so assume not.
```

Comment:
setCI(true) is called at the top of this file - is that not reliable? I feel like either this line or the other isn't what it seems

Reply:
Removed the old comment